### ISSUE TITLE
`named-functions-in-promises` false positives

### ISSUE THREAD

**Issue by SaladFork** [netguru/eslint-plugin-ember#11]
![](https://i.imgur.com/10UggdN.png)

```js
somePromise().then(this.someMethod)
```
will result in a `named-functions-in-promises` linting error, despite using a named function.

**Comment by michalsnik** [comment]
Hey @SaladFork, thanks for pointing this. I'll look into this shortly :)
